pkg/cmd/server/bootstrappolicy/web_console_role_test.go

```diff
@@ -0,0 +1,102 @@
+package bootstrappolicy
+
+import (
+	// standard lib
+
+	"strings"
+	"testing"
+	// kube stuff
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	// openshift stuff
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+)
+
+// NOTE: If this test fails, talk to the web console team to decide if your
+// new role(s) should be visible to an end user in the web console.
+
+var rolesToHide = sets.NewString(
+	"cluster-admin",
+	"cluster-debugger",
+	"cluster-reader",
+	"cluster-status",
+	"registry-admin",
```

Comment:
So if I remove them the test will fail like this:
![screen shot 2017-06-07 at 2 59 01 pm](https://user-images.githubusercontent.com/280512/26896200-f6c68db2-4b91-11e7-9504-2606a5fb5126.png) 

The goal was to make it as easy/painless as possible for someone unfamiliar with the `systemOnly` annotation to just add the string name of their role to either `rolesToShow` or `rolesToHide`, get the test to pass & keep the console clean/free of future surprises.

The only "special" case we made is the `isControllerRole()` function, to make it slightly easier to maintain this list.  I'm not terribly concerned (in this test) where the roles come from, so long as they are correctly hidden/visible in the end.

Do you have a preference for a change here?